prov/efa/src/rdm/efa_rdm_ep_progress.c

```diff
@@ -563,7 +565,15 @@ ssize_t efa_rdm_ep_send_queued_pkts(struct efa_rdm_ep *ep,
 		 */
 		dlist_remove(&pkt_entry->entry);
 
-		ret = efa_rdm_pke_sendv(&pkt_entry, 1);
+		base_hdr = efa_rdm_pke_get_base_hdr(pkt_entry);
+		switch (base_hdr->type) {
```

Comment:
since we only have two allowed types (send and write), we should use if/else and add use assert for safe.

Reply:
We are switching here on packet type. The send packet can be of many different types (eager RTM, medium RTM, long CTS RTM etc), so we cannot do an assert.

I can change the switch to if/else.